### PR TITLE
[eos]: Documentation updates (status, metadata, version, fqcn, removed_at_date)

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### SUMMARY

Modules updated with
- metadata
- version
- fqcn
- removed_at_date (deprecated modules)

### CODE REVIEW

File: plugins/modules/eos_interface.py [hunk 1]
```diff
@@ -104,61 +100,90 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (rohitthakur2590) [inline plugins/modules/eos_interface.py:119]: I think these were unintentional.

**Comment by NilashishC** [comment]
@GomathiselviS is it worth removing the version_added from doc_fragments in this PR?

File: plugins/modules/eos_l3_interface.py [hunk 2]
```diff
@@ -55,40 +52,68 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (rohitthakur2590) [inline plugins/modules/eos_l3_interface.py:82]: I think these are unintentional

File: plugins/modules/eos_linkagg.py [hunk 3]
```diff
@@ -61,47 +58,75 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (rohitthakur2590) [inline plugins/modules/eos_linkagg.py:88]: same as above

File: plugins/modules/eos_vlan.py [hunk 4]
```diff
@@ -78,45 +75,73 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (rohitthakur2590) [inline plugins/modules/eos_vlan.py:103]: same as above

**Pull request merged**